torchmetrics/__about__.py

```diff
@@ -11,7 +11,7 @@
 [PyTorch Lightning](https://pytorch-lightning.readthedocs.io/en/stable/). It was originally a part of
 Pytorch Lightning, but got split off so users could take advantage of the large collection of metrics
 implemented without having to install Pytorch Lightning (even though we would love for you to try it out).
-We currently have around 25+ metrics implemented and we continuously is adding more metrics, both within
+We currently have around 50+ metrics implemented and we continuously is adding more metrics, both within
```

Comment:
```suggestion
We currently have around 60+ metrics implemented and we continuously are adding more metrics, both within
```
58 class like metrics and wrappers so I would round it to 60? as with nxt release we will have at least two more...